src/more-infos/more-info-camera.html

```diff
@@ -29,21 +29,26 @@
     stateObj: {
       type: Object,
     },
+
+    isVisible: {
+      type: Boolean,
+      value: true,
+    },
   },
 
   imageLoaded: function () {
     this.fire('iron-resize');
   },
 
-  computeCameraImageUrl: function (hass, stateObj) {
+  computeCameraImageUrl: function (hass, stateObj, isVisible) {
     if (hass.demo) {
       return '/demo/webcam.jpg';
-    } else if (stateObj) {
+    } else if (stateObj && isVisible) {
       return '/api/camera_proxy_stream/' + stateObj.entityId +
              '?token=' + stateObj.attributes.access_token;
     }
-    // Return an empty image if no stateObj (= dialog not open)
+    // Return an empty image if no stateObj (= dialog not open) or in cleanup mode.
```

Comment:
stateObj can still be null is some flow I didn't change. (i.e. not from dynamicContentUpdater)
In that case camera was using the fallback image and will still do so after my change.